src/test_webdriver_install.py

```diff
@@ -0,0 +1,72 @@
+import os
+from selenium import webdriver
+import requests
+from requests import Response
+
+from webdriver_manager.chrome import ChromeDriverManager
+from webdriver_manager.core.download_manager import WDMDownloadManager
+from webdriver_manager.core.http import HttpClient
+from webdriver_manager.core.logger import log
+from requests.adapters import HTTPAdapter
+
+class CustomHttpClient(HttpClient):
```

Comment:
I suggest we should import `CustomHttpClient` from `asvz_bot.py` to actually test the application code. WDYT?